JavaShared/src/main/java/org/openzen/zenscript/javashared/prepare/JavaPrepareDefinitionVisitor.java

```diff
@@ -131,23 +131,23 @@ public class JavaPrepareDefinitionVisitor implements DefinitionVisitor<JavaClass
 		{
 			JavaClass string = new JavaClass("java.lang", "String", JavaClass.Kind.CLASS);
 			JavaNativeClass cls = new JavaNativeClass(string);
-			cls.addMethod("contains", JavaSpecialMethod.CONTAINS_AS_INDEXOF);
+			//cls.addMethod("contains", JavaSpecialMethod.CONTAINS_AS_INDEXOF);
 			cls.addInstanceMethod("compareToIgnoreCase","compareToIgnoreCase", "(Ljava/lang/String;)I");
 			cls.addInstanceMethod("equalsIgnoreCase","equalsIgnoreCase", "(Ljava/lang/String;)Z");
 
-			cls.addInstanceMethod("indexOfFrom","indexOf", "(II)I");
-			cls.addInstanceMethod("lastIndexOf", "lastIndexOf", "(I)I");
-			cls.addInstanceMethod("lastIndexOfFrom", "lastIndexOf", "(II)I");
-			cls.addInstanceMethod("indexOf", "indexOf", "(I)I");
+			//cls.addInstanceMethod("indexOf", "indexOf", "(I)I");
```

Comment:
Removed for now (check discord) since having them implemented in stdlib and here caused compilation issues...